research/berry/berrylib/test_berry.py

```diff
@@ -192,9 +198,6 @@ def test_fast_inla(method, N=10, iterations=1):
 
     sigma2_post, exceedances, theta_max, theta_sigma = out
 
-    def logistic(x):
-        return jax.scipy.special.expit(x)
-
     # Do our comparisons in probability space.
     np.testing.assert_allclose(
         logistic(theta_max[0, 12]),
```

Comment:
I think the logistic transform is fine here for `theta_max`. It's only a problem for `sigma2_post`. Those values are not in logit space. Those are densities for the sigma2 posterior so it doesn't make sense to logistic them.